### DISCUSSION TITLE
What scale can I take this program to?

### DISCUSSION THREAD

**Discussion by GameSterDamian** [charlesbel/Microsoft-Rewards-Farmer#302] [Q&A]
Is there anything stopping me from buying 1000 emails, adding them to the program, renting a server, and automatically running this every day? Would the program break at this scale? I haven't heard of anyone doing this and I'm wondering why not.

**Comment by udu3324** [comment]
Currently it takes 30-50 minutes per account to do all their daily points. It also takes some cpu and ram to run chrome. It would require a beefy computer to do that much accounts multi threaded. 
Also please don't do that much since they'll see all the traffic and start patching things faster

**Reply by GameSterDamian** [reply]
Don't worry, I wont
